packages/hash/frontend/src/components/hooks/useResolveComment.ts

```diff
@@ -1,13 +1,14 @@
 import { useMutation } from "@apollo/client";
 import { useCallback } from "react";
+import { EntityId } from "@hashintel/hash-subgraph";
 import {
   ResolvePersistedCommentMutation,
   ResolvePersistedCommentMutationVariables,
 } from "../../graphql/apiTypes.gen";
 import { resolvePersistedComment } from "../../graphql/queries/comment.queries";
 import { getPersistedPageComments } from "../../graphql/queries/page.queries";
 
-export const useResolveComment = (pageId: string) => {
+export const useResolveComment = (pageId: EntityId) => {
```

Comment:
Nit:
```suggestion
export const useResolveComment = (pageEntityId: EntityId) => {
```

Reply:
I started to rename these but there are far too many places to update this sort of thing :(